nym-vpn-core/nym-vpn-lib/src/lib.rs

```diff
@@ -42,10 +42,12 @@ pub use nym_task::{
     StatusReceiver,
 };
 
+use crate::platform::set_listener_status;
 #[cfg(any(target_os = "ios", target_os = "macos"))]
 pub use crate::platform::swift;
 #[cfg(target_os = "ios")]
 use crate::platform::swift::OSTunProvider;
+use crate::uniffi_custom_impls::{ExitStatus, StatusEvent};
```

Comment:
I see uniffi bleeding into the main lib. Ideally the dependency should be that the uniffi code depends on the lib, but not the other way?